blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/Export.java

```diff
@@ -158,6 +160,9 @@ public Object getValue(Property property, Object model, ExportConfig config) thr
                     printError(model.getClass(), e);
                     return SKIP;
                 }
+            } else if (model instanceof Item || model instanceof Run) {
```

Comment:
@vivek I have skipped serialisation of `Item` and `Run` objects in case they are returned by a `Cause` object. These are evil for performance reasons anyway.